index.adoc

```diff
@@ -116,6 +116,8 @@ TIP: The benefit of using a Linux package manager to install the gem is that it
 The drawback is that the package may not be available immediately after the release of the gem.
 If you need the latest version, you can always fallback to using the `gem` command.
 
+TIP: If you are working on Windows, you may get this error : `ERROR:  While executing gem ... (Encoding::UndefinedConversionError)`. In that case, you have to change the code page used by your command line. This is done by executing `chcp 1252`. Then use the following commands as stated.
```

Comment:
I like this suggestion, but Asciidoctor strongly recommends the use of UTF-8, so this should be:

```
$ chcp 65001
```